docs/nomad_simulations.md

```diff
@@ -1 +1,499 @@
+    ```python
+    model_system.chemical_formula.normalize(archive=None, logger=logger)
+    ```
+
+    Now, the `ChemicalFormula` sub-section contains the information of the different formats for the chemical formula: `reduced` and `hill` are both `'AsGa'`, while `iupac` is `'GaAs'`. The `descriptive` formula is set depending on the chemical elements present in the system, and because neither `'H'` nor `'O'` is present (i.e., no organic formulas), and gallium arsenide does not belong to any format exceptions, then it is set to the inorganic typical formula `'GaAs'`, coinciding with `iupac`. The `anonymous` formula is `'AB'`.
+
+
+
+## `Outputs` {#outputs}
+
+The `Outputs` section is an output section which contains all the information about the computed properties by the simulation, as well as references to the specific `ModelMethod` and `ModelSystem` used to compute such properties. These properties are under `Outputs` in a long list of sub-sections, and they inherit from an abstract section called `PhysicalProperty`. This section contains the `value` of the property as well as other quantities. The `variables` sub-section allows to write a physical property in term of a varying parameter, and it also allows to calculate the shape of `value` in a dynamic way. This means that, the same physical property (e.g., `ElectronicBandGap`) could have different shapes depending on the problem being parsed (e.g., a single scalar number or a set of varying scalars with respect to a variable, e.g., `Temperature`).
+
+The `Outputs` section can be further specialized into `SCFOutputs` in case the properties are calculated in a series of self-consistent steps. The steps are stored under a sub-section called `scf_steps`, while the last step and other non-self-consistently calculated properties are stored directly under `SCFOutputs`. The reference to the `SelfConsistency` section (see [NumericalSettings](#numericalsettings)) allows us to automatically determine if a self-consistently calculated property is converged or not.
+
+The detailed relationship tree is:
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="../assets/part2-nomad-simulations/outputs.png" alt="Outputs quantities and functions UML diagram." width="80%" title="Click to zoom in">
+    </label>
+</div>
+
+
+!!! abstract "Assignement 2.7"
+    Instantiate a `SCFOutputs` section. We are going to store a self-consistently calculated `FermiLevel`, whose steps where `[1, 1.5, 2, 2.1, 2.101]` in eV. Also add the references to the section `ModelMethod` done in Assignement 2.3 and the section `ModelSystem` done in Assignement 2.6. For the self-consistently calculated `FermiLevel` section, add the reference to the section `SelfConsistency` done in Assignement 2.4.
```

Comment:
"an `SCFOutputs`"

Reply:
"...`FermiLevel`, whose values at each step are `[...]` eV."?

Reply:
"~~Also~~ Add the appropriate references to the `ModelMethod` and `ModelSystem` sections created? in Assignments 2.3 and 2.6, respectively"

Reply:
changed done to "created" or whatever other word you decide for the next sentence as well